crates/turborepo-auth/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(clippy::all)]
```

Comment:
why don't we have this as the default behavior? If we're using a linter, shouldn't we go all in and require that it passes universally?

Reply:
Sadly due to a few constraints we can't just make this the default behavior:
 - Workspace level clippy configuration is [severely limited](https://doc.rust-lang.org/nightly/clippy/lint_configuration.html) to what we can configure at that level
 - Turborepo crates aren't default members of the workspace

We could pass it as part of the invocation during CI so at least CI would always catch it, but it appears that `cargo groups` doesn't support arg forwarding:
```
[2 olszewski@chriss-mbp] /Users/olszewski/code/vercel/turborepo $ cargo groups clippy turborepo-libraries --features rustls-tls -- --deny clippy::all   
error: unexpected argument '--deny' found

Usage: Usage: cargo groups clippy [OPTIONS] <GROUP>

For more information, try '--help'.
```
@NicholasLYang if you're looking for something to do on your DYF 😉